OutOfSchool/OutOfSchool.WebApi/Models/Workshop/WorkshopBaseCard.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using OutOfSchool.Common.Enums;
+
+namespace OutOfSchool.WebApi.Models.Workshop;
```

Comment:
(Code style) Namespace naming is inconsistent. Your `OutOfSchool.WebApi.Models.Workshop` has `Models` (in plural) but `Workshop` (in singular). Please find a uniform way for naming

Reply:
I know, that this was situation before you came here, so you could ignore my comment, but, you know, PR named `Refactoring WorkshopDTOs` is tempting for such requests

Reply:
I agree with you. Fixed.